website/public/main.css

```diff
@@ -718,7 +721,48 @@ kbd.keystroke {
     bottom: 0;
   }
 }
-
+.comparison-mobile {
+  display: none;
+}
+#editors-support {
+  display: none;
+}
+@media only screen and (max-width: 684px) {
+  .comparison {
+    display: none;
+  }
+  .comparison-mobile {
+    display: initial;
+  }
+  .comparison-mobile table {
+    width: 100%;
+    text-align: center;
+    border-collapse: collapse;
+    border: 1px solid gray;
+  }
+  .comparison-mobile table th,
+  td {
+    padding: 0.5em 0em;
+    border: 1px solid gray;
+  }
+  #editors-support {
+    display: initial;
+  }
+  #supported-editors {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
```

Comment:
I would have expected `display: flex;` here. What is the advantage of `display: grid;`?

Reply:
there is no huge advantage, besides that grid requires lines of css to achieve that layout than flex does. Do you want me to change it to flex, I think grid is fine.

Reply:
Grid is fine.